src/Umbraco.Web/PublishedContentQuery.cs

```diff
@@ -191,46 +191,52 @@ public IEnumerable<PublishedSearchResult> Search(string term, string culture = "
         /// <inheritdoc />
         public IEnumerable<PublishedSearchResult> Search(string term, int skip, int take, out long totalRecords, string culture = "*", string indexName = null)
```

Comment:
We can already set the default argument value, so the used index is visible to callers:

```suggestion
         public IEnumerable<PublishedSearchResult> Search(string term, int skip, int take, out long totalRecords, string culture = "*", string indexName = Constants.UmbracoIndexes.ExternalIndexName)
```

The check within the method should be kept, so passing `null` explicitly still uses the external index... Thoughts?

Reply:
This should then also be done on `IPublishedContentQuery`, as the interface is used/visible to callers 😉